Maestro/test/Microsoft.DotNet.Maestro.SubscriptionValidation/packages.config

```diff
@@ -0,0 +1,10 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="xunit" version="2.1.0" targetFramework="net452" />
+  <package id="xunit.abstractions" version="2.0.0" targetFramework="net452" />
+  <package id="xunit.assert" version="2.1.0" targetFramework="net452" />
+  <package id="xunit.core" version="2.1.0" targetFramework="net452" />
+  <package id="xunit.extensibility.core" version="2.1.0" targetFramework="net452" />
+  <package id="xunit.extensibility.execution" version="2.1.0" targetFramework="net452" />
+  <package id="xunit.runner.msbuild" version="2.1.0" targetFramework="net461" />
```

Comment:
Right; the project would need `xunit.runner.visualstudio` for them to be visible in Test Explorer. We can add that if desired, but I think it would be unnecessary overhead at this point.